src/wp-includes/class-wp-duotone.php

```diff
@@ -948,45 +944,45 @@ public static function register_duotone_support( $block_type ) {
 	 * This handles selectors defined in `color.__experimentalDuotone` support
 	 * if `filter.duotone` support is not defined.
 	 *
-	 * @param string $block_name The block name.
+	 * @param WP_Block_Type $block_type Block type to check for support.
 	 *
 	 * @internal
 	 *
 	 * @since 6.3.0
 	 *
 	 * @return string The CSS selector or null if there is no support.
 	 */
-	private static function get_selector( $block_name ) {
-		$block_type = WP_Block_Type_Registry::get_instance()->get_registered( $block_name );
-
-		if ( $block_type && property_exists( $block_type, 'supports' ) ) {
-			/*
-			 * Backwards compatibility with `supports.color.__experimentalDuotone`
-			 * is provided via the `block_type_metadata_settings` filter. If
-			 * `supports.filter.duotone` has not been set and the experimental
-			 * property has been, the experimental property value is copied into
-			 * `supports.filter.duotone`.
-			 */
-			$duotone_support = _wp_array_get( $block_type->supports, array( 'filter', 'duotone' ), false );
-			if ( ! $duotone_support ) {
-				return null;
-			}
+	private static function get_selector( $block_type ) {
+		if ( ! ( $block_type instanceof WP_Block_Type ) ) {
+			return null;
+		}
 
-			/*
-			 * If the experimental duotone support was set, that value is to be
-			 * treated as a selector and requires scoping.
-			 */
-			$experimental_duotone = _wp_array_get( $block_type->supports, array( 'color', '__experimentalDuotone' ), false );
-			if ( $experimental_duotone ) {
-				$root_selector = wp_get_block_css_selector( $block_type );
-				return is_string( $experimental_duotone )
-					? WP_Theme_JSON::scope_selector( $root_selector, $experimental_duotone )
-					: $root_selector;
-			}
+		/*
+		 * Backwards compatibility with `supports.color.__experimentalDuotone`
```

Comment:
According to https://make.wordpress.org/core/handbook/glossary/

```suggestion
		 * Backward compatibility with `supports.color.__experimentalDuotone`
```